lib/seccomp-tools/cli/dump.rb

```diff
@@ -56,14 +61,26 @@ def parser
       def handle
         return unless super
 
-        option[:command] = argv.shift unless argv.empty?
-        SeccompTools::Dumper.dump('/bin/sh', '-c', option[:command], limit: option[:limit]) do |bpf, arch|
+        block = lambda do |bpf, arch|
           case option[:format]
           when :inspect then output { '"' + bpf.bytes.map { |b| format('\\x%02X', b) }.join + "\"\n" }
           when :raw then output { bpf }
           when :disasm then output { SeccompTools::Disasm.disasm(bpf, arch: arch) }
           end
         end
+        if option[:pid].nil?
+          option[:command] = argv.shift unless argv.empty?
+          SeccompTools::Dumper.dump('/bin/sh', '-c', option[:command], limit: option[:limit], &block)
+        else
+          begin
+            SeccompTools::Dumper.dump_by_pid(option[:pid], option[:limit], &block)
+          rescue Errno::EPERM, Errno::EACCES => e
+            warn(e)
+            warn('PTRACE_SECCOMP_GET_FILTER requires CAP_SYS_ADMIN')
+            warn("Try this: sudo seccomp-tools dump -p #{option[:pid]} -l #{option[:limit]}")
```

Comment:
replace the option here to be exactly what the user passed to prevent confusing the user